pkg/admission/apibinding/apibinding_admission.go

```diff
@@ -96,6 +97,19 @@ func (o *apiBindingAdmission) Admit(ctx context.Context, a admission.Attributes,
 		apiBinding.Spec.Reference.Workspace.Path = cluster.Name.String()
 	}
 
+	// set labels
+	if apiBinding.Spec.Reference.Workspace == nil {
+		delete(apiBinding.Labels, apisv1alpha1.InternalAPIBindingExportLabelKey)
+	} else {
+		if apiBinding.Labels == nil {
+			apiBinding.Labels = make(map[string]string)
+		}
+		apiBinding.Labels[apisv1alpha1.InternalAPIBindingExportLabelKey] = permissionclaims.ToAPIBindingExportLabelValue(
```

Comment:
when is this applicable? it seems like it always be overwritten by the controller to `ToReflexiveAPIBindingLabelKeyAndValue`

Reply:
ah, okay, we use two different keys